hw/block/m25p80.c

```diff
@@ -631,31 +631,33 @@ void flash_write8(Flash *s, uint32_t addr, uint8_t data)
     uint32_t page = addr / s->pi->page_size;
     uint8_t prev = s->storage[s->cur_addr];
 
-    if (!s->write_enable) {
-        qemu_log_mask(LOG_GUEST_ERROR, "M25P80: write with write protect!\n");
-        return;
-    }
```

Comment:
How come this was moved?

Reply:
Upstream review stipulates that the declarations should be at the top of the function. So, rather, I moved the block_protect_value declaration above this